code/datums/limb.dm

```diff
@@ -415,6 +415,76 @@
 		user.lastattacked = target
 		ON_COOLDOWN(src, "limb_cooldown", COMBAT_CLICK_DELAY)
 
+
+
+/datum/limb/maneater_mouth
+	var/sound_attack = 'sound/voice/animal/short_hiss.ogg'
+	var/dam_low = 8
+	var/dam_high = 12
+	var/custom_msg = null
+	var/miss_prob = 75
+	var/stam_damage_mult = 0
+	var/human_stam_damage = 50
+	var/human_desorient_duration = 2 SECONDS
+	var/human_stun_duration = 5 SECONDS
+	var/list/chems_to_inject = null
+	var/amount_to_inject = 3
+
+/datum/limb/maneater_mouth/New(var/obj/item/parts/holder)
+	..()
+	src.chems_to_inject = list()
+
+/datum/limb/maneater_mouth/attack_hand(atom/target, var/mob/user, var/reach)
+	if (ismob(target))
+		..()
+
+	if (isitem(target))
+		var/obj/item/potentially_food = target
+		if (potentially_food.edible)
+			potentially_food.attack(user, user)
+
+/datum/limb/maneater_mouth/help(mob/target, var/mob/user)
+	return
+
+/datum/limb/maneater_mouth/disarm(mob/target, var/mob/user)
+	return
+
+/datum/limb/maneater_mouth/grab(mob/target, var/mob/user)
+	return
+
+/datum/limb/maneater_mouth/harm(mob/target, var/mob/user)
+	if (!user || !target)
+		return 0
+
+	if (!target.melee_attack_test(user))
+		return
+
+	if (prob(src.miss_prob) || is_incapacitated(target)|| target.restrained())
+
+		var/datum/attackResults/msgs = user.calculate_melee_attack(target, dam_low, dam_high, 0, stam_damage_mult, !isghostcritter(user), can_punch = 0, can_kick = 0)
+		user.attack_effects(target, user.zone_sel?.selecting)
+		if (isliving(target) && !issilicon(target))
+			var/mob/living/victim = target
+			//we want to stun the target long enough to get grabbed in find themselves about to get eaten, but not long enough to not be able to have the chance to struggle out of the grab
+			victim.do_disorient(src.human_stam_damage, paralysis = src.human_stun_duration, disorient = src.human_desorient_duration, stack_stuns = FALSE)
+			//after the stun, as a little treat for skilled botanist, a maneater that got splices in it tries to inject its victims
+			if (length(src.chems_to_inject) > 0)
+				var/chem_protection = 0
+				if (ishuman(victim))
+					chem_protection = ((100 - victim.get_chem_protection())/100) //not gonna inject people with bio suits (1 is no chem prot, 0 is full prot for maths)
+				var/injected_per_reagent = max(0.1 , chem_protection * src.amount_to_inject / length(src.chems_to_inject))
+				if (injected_per_reagent > 0.1)
+					for (var/plantReagent in src.chems_to_inject)
+						victim.reagents?.add_reagent(plantReagent, injected_per_reagent)
+
+		msgs.base_attack_message = src.custom_msg ? src.custom_msg : "<b><span class='combat'>[user] bites [target]!</span></b>"
+		msgs.played_sound = src.sound_attack
+		msgs.flush(0)
+	else
+		user.visible_message("<b><span class='combat'>[user] attempts to bite [target] but misses!</span></b>")
+	user.lastattacked = target
+	ON_COOLDOWN(src, "limb_cooldown", COMBAT_CLICK_DELAY)
```

Comment:
while i agree, doubling the cooldown is a bit more than i think would be feaseable. Would need to test it, from my tests they weren't able to grab a monkey with a 4.5 second stun, and i'm not sure they will be able with a higher combat delay, so maybe if i change that i would to slightly raise the delay as well. I test around with some numbers and look what feels right.

Reply:
Alright, so with 1.5 second delay it was alright. with 2 second delay (maybe because of lagg of my potato, no idea) the AI was so slow that, without stam crits, the maneater was unable to stamdrain a target dummy. I have increased the combat delay to 1.5 seconds now and reduced the miss chance and upped the stam drain slightly to 60 to make deal less damage but still drain stamina effectively enough.

Reply:
ok, some more tests. This time with me trying to just move away from it. With the delay and the slow speed of the maneater, you are 100% guaranteed to never get downed if you just move away. Especially with how slow they move. The problem is them just not moving after the attack delay